docs/content/manual/manual.yml

```diff
@@ -3075,13 +3075,29 @@ sections:
           to invoke jq with the -n command-line option, otherwise
           the first entity will be lost.
 
-      - title: "`debug`"
+      - title: "`debug`", "`debug(msgs)`"
         body: |
 
-          Causes a debug message based on the input value to be
-          produced.  The jq executable wraps the input value with
-          `["DEBUG:", <input-value>]` and prints that and a newline on
-          stderr, compactly.  This may change in the future.
+          These two filters are like `.` but have as a side-effect the
+          production of one or more messages on stderr.
+          
+          The message produced by the `debug` filter has the form
+          `["DEBUG:", <input-value>]` where <input-value> is a compact rendition of 
```

Comment:
There is an extra space after `of` at the end of this line, it will result in a double space in the manual when the site/manpage are generated, so it should be removed

Reply:
Thanks for the hint about the extra space.  As for jq.1.prebuilt - I've tried many ways and many times to get the python scripts to work, but with no success.  E.g. just now, after successfully installing lxml:

$ pip3 list | grep lxml
lxml              4.9.2

$ pip list | grep lxml
lxml              4.9.2

$ pipenv run ./build_manpage.py
ModuleNotFoundError: No module named 'lxml'

$ pipenv run python3 build_manpage.py
ModuleNotFoundError: No module named 'lxml'

# Similar problems with man.test, by the way:
$ pipenv run ./build_mantests.py
ModuleNotFoundError: No module named 'yaml'

It had been my impression that updating these downstream files could be postponed e.g. until the site was regenerated.  But again, please feel free.

Reply:
I think you just need to run `(cd ./docs && pipenv install)` then run `./configure --enable-docs #something else if you want`, and finlly you should be able to just run `make jq.1.prebuilt`.

Reply:
Thanks, that worked. `man.test` did not change.  It's missing some tests though.

Any hints on how to avoid the Oniguruma submodule problem when attempting to "squash" commits?   
(GitHub Desktop says: "Error: Unable to squash when changes are present on your branch. The following files would be overwritten: modules/onigurma "

Reply:
@pkoppstein 
I think you just need to run `git submodule update`; you have probably pulled new commits in `modules/oniguruma` without running that.

Reply:
@emanuele6 - Thanks, that worked. I hope that the squashed commit won't cause anyone any grief.

Btw, I am quite sure that I didn't do anything (deliberately or otherwise) to the modules/oniguruma repository apart from whatever was done when I first cloned jqlang/jq.  (I did notice that "modules/oniguruma" was marked as having changed when I created the clone.)

Reply:
Ah, maybe then you have not run `git submodule update` after pulling 7c1efd6 (2 weeks ago) from master.
If you don't run `git pull --recurse-submodule`, or don't have the `submodule.recurse` gitconfig setting set to `true`, you need to do that after you pull.

Reply:
@pkoppstein I just to pull 092fef7 (that upgrades `modules/oniguruma`), with just `git pull` without `submodule.recurse` set to true, and indeed I needed to run `git submodule update` manually.